web/modules/custom/dpl_library_token/src/LibraryTokenHandler.php

```diff
@@ -57,34 +53,42 @@ class LibraryTokenHandler {
    *   The key value expire keyValueFactory.
    * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
    *   Configuration.
-   * @param \GuzzleHttp\ClientInterface $http_client
+   * @param \GuzzleHttp\ClientInterface $httpClient
    *   The HTTP client.
    * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $logger
    *   The library token logger channel.
+   *
+   * @throws \Drupal\dpl_library_token\Exception\MissingConfigurationException
+   * @throws \Safe\Exceptions\StringsException
    */
   public function __construct(
     KeyValueExpirableFactoryInterface $keyValueFactory,
     ConfigFactoryInterface $configFactory,
-    ClientInterface $http_client,
+    protected ClientInterface $httpClient,
```

Comment:
Constructor property promotion is not something we use elsewhere. Personally I am not sure how I feel about it outside of DTOs. In the current form it somewhat bypasses requirements regarding documentation of class properties mentioned above. Perhaps that is in fact a good thing.

If we want it we should consider [requiring it with our coding standard](https://github.com/slevomat/coding-standard#slevomatcodingstandardclassesrequireconstructorpropertypromotion-). If not we should consider preventing it using the same tools.